packages/drivers/odsp-driver/src/odspSnapshotParser.ts

```diff
@@ -62,11 +62,16 @@ export function convertOdspSnapshotToSnapshotTreeAndBlobs(
             blobsWithBufferContent.set(blob.id, stringToBuffer(blob.content, blob.encoding ?? "utf8"));
         });
     }
+
+    const sequenceNumber = odspSnapshot?.trees[0].sequenceNumber;
+
     const val: ISnapshotContents = {
         blobs: blobsWithBufferContent,
         ops: odspSnapshot.ops?.map((op) => op.op) ?? [],
-        sequenceNumber: odspSnapshot?.trees[0].sequenceNumber,
+        sequenceNumber,
         snapshotTree: buildHierarchy(odspSnapshot.trees[0]),
+        latestSequenceNumber: (odspSnapshot.ops && odspSnapshot.ops.length > 0) ?
```

Comment:
@GaryWilber - Looking at this code for the first time, question for you - is `odspSnapshot.ops` always changing over time? In other words, every time you fetch the snapshot, you get not only the latest snapshot, but also all the ops up to that moment, so you could call twice and get the same `snapshotTree` and `sequenceNumber` but different values for `latestSequenceNumber`, right?

Reply:
Correct. `latestSequenceNumber` will be the sequence number of the latest op in SPO.
Ops can keep being posted to SPO and so that number will keep changing, even though the latest summary is still the same.